src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2310,6 +2310,26 @@ void processContextElement
                     getEntitiesCollectionName(tenant).c_str(),
                     query.toString().c_str()));
 
+    std::map<std::string, std::string>::iterator it = uriParams.find("op");
+
+    // if is an append and uri has not possibility of options
+    if (strcasecmp(action.c_str(), "append") == 0 && it == uriParams.end())
+    {
+        EntityIdVector vec;
+        vec.push_back(enP);
+
+        AttributeList attList;
+        Restriction res;
+        ContextElementResponseVector rsp;
+        std::string err;
+        entitiesQuery(vec, attList, res, &rsp, &err, true, tenant, servicePathV);
```

Comment:
After thinking on this a little bit, actually it should be an excesive overhead, as this check is only triggered when ?op=append / APPEND_STRICT is used, which I supposed will not be very often.

Thus, I think we can progress this PR with the current strategy. Improving that has been deferred to a tech debt issue (https://github.com/telefonicaid/fiware-orion/issues/1207).

NTC (=Nothing To Change, regarding this comment thread).